glide.yaml

```diff
@@ -38,7 +38,8 @@ import:
 
 # openshift second
 - package: github.com/openshift/api
-  version: release-3.10
+  # the commit below points to release-3.10 branch
+  version: 6ef6704e604af539af656171d5c0bde774923a43
```

Comment:
```
// point to working openshift/api commit
// TODO(network): fix openshift/api network types
```

Please create a prio urgent 3.10 BZ to the fix the API types and assign to @squeed.

Reply:
done - https://bugzilla.redhat.com/show_bug.cgi?id=1755306